main.tf

```diff
@@ -94,18 +96,23 @@ resource "helm_release" "castai_evictor" {
   namespace       = "castai-agent"
   create_namespace = true
   cleanup_on_fail = true
+  wait = true
```

Comment:
wait is true by default. but we can be explicit since we need to have evictor in snapshot